### PR TITLE
Rtw workspace port config

### PR DESCRIPTION
**Pull request by muritane**

### CODE REVIEW

**Comment by destogl** [comment]
Have you tested here the concept we were talking about to set up environment variables?

I would like to test this, but without variable export workspaces could stop to work. Is this correct?

**Comment by muritane** [comment]
> Have you tested here the concept we were talking about to set up environment variables?
> 
> I would like to test this, but without variable export workspaces could stop to work. Is this correct?

now it can be tested, the env variables are exported via file that is being checked similar to PS1:
https://github.com/StoglRobotics/ros_team_workspace/pull/141/files#diff-ab55cd33c997f1accf14a7afa9338e405fff5c46346b920a421dfbe1d1e0f21bR27

File: rtwcli/rtw_cmds/rtw_cmds/workspace/verbs.py [hunk 1]
```diff
@@ -12,11 +12,374 @@
...
 from rtwcli.verb import VerbExtension
+from send2trash import send2trash
+
+WORKSPACES_PATH = os.path.expanduser("~/.ros_team_workspace/workspaces.yaml")
+USE_WORKSPACE_SCRIPT_PATH = os.path.expanduser(
+    "~/ros_team_workspace/scripts/environment/setup.bash"
+)
+WORKSPACES_KEY = "workspaces"
+ROS_TEAM_WS_RC_PATH = os.path.expanduser("~/.ros_team_ws_rc")
+WORKSPACES_PATH_BACKUP = os.path.expanduser("~/.ros_team_workspace/workspaces_backup.yaml")
```
Comment (destogl) [inline rtwcli/rtw_cmds/rtw_cmds/workspace/verbs.py:64]: We should work with backups similarly as Manual programmed, this by adding multiple versions of the files. Maybe adding those into a subfolder to not clutter the main folder would be good. We can then think about removing those files later if necessary.

File: rtwcli/README.md [hunk 2]
```diff
@@ -7,3 +7,15 @@
...
+
+## porting workspace
+
+add explicit export to variables: `bash ros_team_workspace/scripts/update-rtw.bash`
+
+source new methods with exports: `source ~/.ros_team_ws_rc`
+
+source your workspace: `_<ws_name>`
+
+port config: `rtw wokspace port`
```
Comment (destogl) [inline rtwcli/README.md:19]: When I try to execute this I get the following error:

```
Failed to load entry point 'port': No module named 'questionary'
Traceback (most recent call last):
File "/usr/local/bin/rtw", line 33, in <module>
sys.exit(load_entry_point('rtwcli', 'console_scripts', 'rtw')())
File "/opt/RosTeamWS/ros_team_workspace/rtwcli/rtwcli/rtwcli/cli.py", line38, in main
add_subparsers_on_demand(
File "/opt/RosTeamWS/ros_team_workspace/rtwcli/rtwcli/rtwcli/command/__init__.py", line 189, in add_subparsers_on_demand
extension.add_arguments(command_parser, f"{cli_name} {name}", **kwargs)
File "/opt/RosTeamWS/ros_team_workspace/rtwcli/rtw_cmds/rtw_cmds/commands.py", line 27, in add_arguments
add_subparsers_on_demand(parser, cli_name, "_verb", self.verbs_group, required=False)
File "/opt/RosTeamWS/ros_team_workspace/rtwcli/rtwcli/rtwcli/command/__init__.py", line 176, in add_subparsers_on_demand
extension = command_extensions[name]
KeyError: 'port'
```

Do you have any idea why? (The same for `port_all`)

**Review by destogl — APPROVED** [review]
Thanks! Porting works good!

**Pull request merged**